frontend/src/page/profile.tsx

```diff
@@ -54,7 +54,8 @@ export const Profile: React.FC = () => {
           const profileUpdateFetchPromise = abortableJsonFetch<StoredUser>(
             API_PROFILE_UPDATE,
             Auth.Required,
-            user
+            user,
+            {method: "POST"}
```

Comment:
Although its technically a POST, the params are still being sent through the query string, which strikes me as a bit odd. :sweat_smile: 
I guess it works and I can't think of any problems this will cause. We will probably want to switch to using form data or something when we start sending profile avatar images.

Reply:
True, I'd love to make it so those expectStringParam type functions could also pull from form data.

Reply:
Notes about cloudflare workers and form data:
https://community.cloudflare.com/t/worker-formdata-get-file-instance/155009

https://community.cloudflare.com/t/cannot-seem-to-send-multipart-form-data/163491